core/src/main/java/io/grpc/transport/AbstractClientStream.java

```diff
@@ -125,8 +130,8 @@ protected void inboundHeadersReceived(Metadata.Headers headers) {
    *
    * @param frame the received data frame. Its ownership is transferred to this method.
    */
-  protected void inboundDataReceived(ReadableBuffer frame) {
-    Preconditions.checkNotNull(frame, "frame");
```

Comment:
@ejona86 I guess I'm failing to see how changing a few methods to use static imports warrants a discussion at all.  Reducing the amount of code while maintaining understandability seems like a win-win to me. I think the likelihood of a ping pong between opposing developers is rather low ... certainly low enough that we don't have to be concerned by a static import.